src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/ComplexCompositeType.cs

```diff
@@ -0,0 +1,275 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace WcfService
+{
+    [DataContract(Namespace = "http://www.contoso.com/wcfnamespace")]
+    public class ComplexCompositeType : IEquatable<ComplexCompositeType>
```

Comment:
What is your plan for files that need to be shared by IIS, SelfHost and test client?  At one point I proposed a separate folder (SharedTestTypes?) reserved for service interface types and data types that both server and client side need to use.  If we can collect them into a common folder, it would be easier to link to them.  Not strictly blocking for this PR, but we need a solution by the time SelfHost and Client code PR's start arriving.